glide.lock

```diff
@@ -1,5 +1,5 @@
 hash: bc512f42285c0afbef5adad730e5bbf97f7581f4e859b13bc9505c285d3f8c42
-updated: 2016-05-10T16:25:42.006454568Z
+updated: 2016-05-25T17:50:10.881650693-04:00
```

Comment:
I'd remove this from the commit since the lockfile doesn't change at all.

BTW, newer versions of glide (perhaps it's only the development version right now) won't modify the last updated timestamp if nothing has changed during an update.